packages/api-clients/react-query/exchange/src/mutator/customInstance.ts

```diff
@@ -0,0 +1,18 @@
+import Axios, { AxiosRequestConfig } from 'axios';
+
+export const AXIOS_INSTANCE = Axios.create({ baseURL: '' });
```

Comment:
I think here as a temporary solution we should add `process.env.BACKEND_URL+process.env.BACKEND_PORT` or new single line `BACKEND_URL` env

Reply:
I agree. I can imagine a situation when someone needs to expose backend API under some path. It is not possible to specify it now with these two variables. But, I'd introduce this change separately.